db/migrate/20130904010950_move_admin_notes_to_comments.rb

```diff
@@ -9,9 +9,14 @@ def self.up
     add_index     :active_admin_comments, [:author_type, :author_id]
 
     # Update all the existing comments to the default namespace
-    say "Updating any existing comments to the #{ActiveAdmin.application.default_namespace} namespace."
-    comments_table_name = ActiveRecord::Migrator.proper_table_name("active_admin_comments")
-    execute "UPDATE #{comments_table_name} SET namespace='#{ActiveAdmin.application.default_namespace}'"
+    $LOADED_FEATURES.each do |f|
+      fname = f.split('/').last
+      if fname == "activeadmin.rb"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.